specs/signature-specification.md

```diff
@@ -261,9 +261,15 @@ The client implementation can use the aforementioned `mediaType` to parse the si
 
 ## Appendix
 
-### Signing time
+### Signing time & Authentic Signing time
 
-The signing time denotes the time at which the signature was generated. A X509 certificate has a defined [validity](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.5) during which it can be used to generate signatures. The signing time must be greater than or equal to certificate's `notBefore` attribute, and signing time must be less than or equal to certificate's `notAfter` attribute. Signatures generated after the certificate expires are considered invalid. An authentic timestamp, like TSA countersignature, allows a verifier to determine if the signature was generated when the certificate was valid. It also allows a verifier to determine if a signature be treated as valid when a certificate is revoked, if the certificate was revoked after the signature was generated. In the absence of an authentic timestamp, signatures are considered invalid after certificate expires, and all signatures are considered revoked when a certificate is revoked.
+A `signing time` denotes the time at which the signature was generated. A X509 certificate has a defined [validity](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.5) during which it can be used to generate signatures. The signing time must be greater than or equal to certificate's `notBefore` attribute, and the signing time must be less than or equal to certificate's `notAfter` attribute. Signatures generated outside the certificate's validity are considered invalid. 
+
+An `authentic signing time` is a signing time authenticated by a trusted third party. It allows a verifier to determine if the signature was generated within the certificate's validity. Notary project supports two such trusted third parities. They are timestamp authority (TSA) and signing authority (SA).
```

Comment:
```suggestion
An `authentic signing time` is a signing time authenticated by a trusted third party. It allows a verifier to determine if the signature was generated within the certificate's validity. Notary Project supports two such trusted third parities. They are timestamp authority (TSA) and signing authority (SA).
```

Reply:
Also, should we use capitalized format in the first letter of each term? See https://datatracker.ietf.org/doc/html/rfc3628#section-3.2

- Time-Stamping Authority (TSA) 
- Signing Authority (SA)